docs/src/citation.md

```diff
@@ -0,0 +1,11 @@
+# Citation
+
+If your work uses this Element, please cite the following manuscript and Research
+Resource Identifier (RRID).
+
++ Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R, Sasaki M, Sitonic D,
+  Reimer J, Walker EY, Tolias AS. DataJoint Elements: Data Workflows for
+  Neurophysiology. bioRxiv. 2021 Jan 1. doi: https://doi.org/10.1101/2021.03.30.437358
+
++ DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) -
```

Comment:
Please add [automated versioning to the citation page](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2)